src/Cake.Core/IO/LaunchInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Cake.Core.IO
+{
+    /// <inheritdoc/>
+    public sealed class LaunchInfo : ILaunchInfo
```

Comment:
Let's also move this file to src/Cake.Core to be side-by-side with CakeEnvironment which is the class that uses it